agenda-services/src/main/java/org/exoplatform/agenda/util/Utils.java

```diff
@@ -389,7 +389,7 @@ public static boolean canInviteeEdit(IdentityManager identityManager,
       return false;
     } else if (StringUtils.equals(SpaceIdentityProvider.NAME, requestedOwner.getProviderId())) {
       Space space = spaceService.getSpaceByPrettyName(requestedOwner.getRemoteId());
-      return space != null && (space.getRedactors() == null || space.getRedactors().length == 0);
+      return space != null && space.getRedactors() != null && space.getRedactors().length > 0;
```

Comment:
Could space managers edit events ?

Reply:
> Could space managers edit events ?

In fact no, I have a problem when I remove a redactor and try to change the status of button toggle.But I don't understand what is redactor's income in enable or disable of this button ??!!